packages/next/src/bin/next.ts

```diff
@@ -105,6 +105,10 @@ program
     )}`
   )
   .option('-d, --debug', 'Enables a more verbose build output.')
+  .option(
+    '--debug-memory-usage',
```

Comment:
```suggestion
    '--experimental-debug-memory-usage',
```
We might wanna start with this being experimental while we iterate on it

Reply:
Done.